packages/rocketchat-lib/server/methods/startEmailIntercepter.js

```diff
@@ -0,0 +1,46 @@
+Meteor.methods({
+	startEmailIntercepter() {
+		if (RocketChat.settings.get('Direct_Reply_Enable') && RocketChat.settings.get('Direct_Reply_Protocol') && RocketChat.settings.get('Direct_Reply_Host') && RocketChat.settings.get('Direct_Reply_Port') && RocketChat.settings.get('Direct_Reply_Username') && RocketChat.settings.get('Direct_Reply_Password')) {
+			if (RocketChat.settings.get('Direct_Reply_Protocol') === 'IMAP') {
+				// stop already running instance
+				if (RocketChat.IMAP && RocketChat.IMAP.isActive()) {
+					RocketChat.IMAP.stop(Meteor.bindEnvironment(function() {
+						RocketChat.IMAP = new RocketChat.IMAPIntercepter({
+							user: RocketChat.settings.get('Direct_Reply_Username'),
```

Comment:
Why not get the config from settings inside the `RocketChat.IMAPIntercepter` constructor? Then you don't need to duplicate the code to configure